src/components/Navigator.vue

```diff
@@ -80,6 +81,10 @@ export default {
       type: Boolean,
       default: false,
     },
+    breakpoint: {
+      type: String,
+      default: '',
```

Comment:
We should probably either make this prop optional or make the default a real value using one of the `BreakpointName` values.